src/main/webapp/app/exam/shared/events/exam-live-event.component.html

```diff
@@ -43,10 +43,6 @@
                         <th>{{ 'artemisApp.studentExamDetail.matriculationNumber' | artemisTranslate }}</th>
                         <td>{{ event.user?.visibleRegistrationNumber }}</td>
                     </tr>
-                    <tr *ngIf="examAttendanceCheckEvent.text && examAttendanceCheckEvent.text.length > 0">
-                        <th>{{ 'artemisApp.exam.events.messages.examAttendanceCheck.comment' | artemisTranslate }}</th>
```

Comment:
Please also remove the translations and the corresponding client test:
<img width="564" alt="image" src="https://github.com/ls1intum/Artemis/assets/9954674/669f7df1-229f-45a6-a26b-8e9d8d4e4f23">

Reply:
Thanks for pointing it out - I did not double check the tests as I would have expected the Client Test Job to fail in such a case.

FYI: The test did not work as intended because the div contains `Attendance Check` in the headline.